styles.css

```diff
@@ -0,0 +1,824 @@
+}
+
+.about-img-courses li p {
+  z-index: 1;
+  font-weight: bold;
+}
+
+.para {
+  position: relative;
+  border: 2px solid white;
+  max-width: 45rem;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  background-color: #c4c4c4;
+  font-weight: 600;
+}
+
+.showcase h2 {
+  position: relative;
+  font-weight: 800;
+  padding-top: 2rem;
+  letter-spacing: 0.2rem;
+}
+
+.para1 {
+  position: relative;
+  max-width: 100%;
+  font-weight: 600;
+  line-height: 1.7;
+  margin-top: 1rem;
+  padding: 2rem;
+  color: #e85242;
+  font-size: 2rem;
+  padding-left: 0;
+}
+
+.program {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-color: #262930;
+  color: #fff;
+  padding: 1rem 0 0 0;
+}
+
+.courses {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 5rem 0 5rem;
+  height: 100%;
+}
+
+.pro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 0.1rem;
+  background-color: #3c3f45;
+  color: #fff;
+}
+
+.img-logo {
+  width: 3rem;
+  align-self: center;
+}
+
+hr {
+  background-color: rgb(250, 19, 19);
+  font-weight: bolder;
+  width: 3rem;
+}
+
+.pro:hover {
+  border: 1px solid #fff;
+}
+
+u {
+  padding: 1rem 0 5rem 0;
+}
+
+.footer1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+  color: #fff;
+  padding: 2.5rem;
+  max-width: 100%;
+}
+
+.footer-logos {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-logos img {
+  margin: 0 0.5rem;
+}
+
+.footer2-img img {
+  max-width: 30%;
+}
+
+.footer-about {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #2e2e2e;
+  padding: 3.5rem 0;
+  max-width: 100%;
+  color: #fff;
+}
+
+.footer2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #d7d7d7;
+  max-width: 100%;
+}
+
+.footer2-img {
+  display: flex;
+  align-items: center;
+  margin: 0.5rem 0 0 1rem;
+  max-width: 100%;
+}
+
+.footer-desc {
+  margin-right: 4rem;
+}
+
+.footer2-img h6 {
+  font-weight: bolder;
+}
+
+.join-us {
+  display: none;
+}
+
+.speakers-group {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  max-width: 100%;
+
+  /* margin: 0 4rem; */
+  justify-content: center;
+  align-items: center;
+}
+
+.speaker-job {
+  font-size: 0.8rem;
+}
+
+.speakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  margin: 0 5rem;
+  padding-top: 1rem;
+}
+
+.speaker-desc {
+  font-size: 0.8rem;
+  background-color: #d7d7d7;
+}
+
+.speaker-name {
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.speaker-group img {
+  max-width: 7rem;
+  height: 7rem;
+  margin: 0 1rem 4rem 0;
+}
+
+.speaker-hr {
+  background-color: #3c3f45;
+}
+
+.speaker-group {
+  display: flex;
+  align-items: center;
+}
+
```

Comment:
- [ ] I can see a great effort trying to match the speaker design as advised earlier by the previous reviewer. However, there are some yet to be addressed.
design given:
![image](https://user-images.githubusercontent.com/68167810/133751052-78e43040-e0e7-4213-bbec-3942d2464d99.png)

your design:
![image](https://user-images.githubusercontent.com/68167810/133751159-57696a08-13eb-4aa3-9eda-704dfe0d9538.png)
- [ ] The text below the speaker's name is of a different color and also the text beneath the `<hr>` does not require the background color as clearly seen from the design.